src/common/mfu_daos.c

```diff
@@ -14,6 +14,10 @@
 #include <gurt/common.h>
 #include <gurt/hash.h>
 
+#ifdef HDF5_SUPPORT
+#include <hdf5.h>
+#endif
+
```

Comment:
@dsikich I think this should be compatible with different versions, along with calling H5Sencode1  instead of H5Sencode in the code.
```suggestion
#ifdef HDF5_SUPPORT
#include <hdf5.h>
#if H5_VERS_MAJOR == 1 && H5_VERS_MINOR < 12
#define H5Sencode1 H5Sencode
#endif
#endif

```